### PR TITLE
Include node version range in node engine warning message

### PR DESCRIPTION
**Pull request by jeffsee55**
Before this change the warning doesn't show what the resolved version of node actually is. Which is slightly confusing when you have an engine rule like `>=18` and the resolved version is `20x`.

```
Warning: Due to "engines": { "node": ">=18" } in your `package.json` file, the Node.js Version defined in your Project Settings ("18.x") will not apply, Node.js Version "20.x" will be used instead. Learn More: http://vercel.link/node-version
```

### CODE REVIEW

File: packages/build-utils/src/fs/run-user-scripts.ts [hunk 1]
```diff
@@ -259,6 +259,7 @@ export async function getNodeVersion(
     !packageJsonVersion,
     availableVersions
   );
+  console.log({ supportedNodeVersion });
```
Comment (TooTallNate) [inline packages/build-utils/src/fs/run-user-scripts.ts]: Remove?

```suggestion
```

File: packages/remix/defaults/server-node.mjs [hunk 2]
```diff
@@ -4,7 +4,7 @@ import {
   installGlobals,
 } from '@remix-run/node';
 
-installGlobals();
+// installGlobals();
```
Comment (TooTallNate) [inline packages/remix/defaults/server-node.mjs:7]: Remove?

Comment (TooTallNate) [inline packages/remix/defaults/server-node.mjs]: Revert?

```suggestion
installGlobals();
```

**Review by TooTallNate — DISMISSED** [review]

File: packages/remix/defaults/server-node.mjs (hunk 2 above)
Comment (jeffsee55) [reply packages/remix/defaults/server-node.mjs]: Sorry!

**Review by erikareads — APPROVED** [review]

**Pull request merged**